adlarge/fixtures-documentation-bundle/1.1/config/routes/dev/adlarge_fixtures_documentation.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;
+use function Symfony\Component\DependencyInjection\Loader\Configurator\service;
+
+return static function (RoutingConfigurator $routingConfigurator): void {
+    $routingConfigurator->import(service('AdlargeFixturesDocumentationBundle/Resources/config/routing.yml'));
+};
```

Comment:
False configuration related to: https://github.com/symplify/symplify/pull/4289/files